tasks/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: Stop OpenTripPlanner
```

Comment:
On a fresh provision, this task fails because there is to `otp` service definition. You may need to do something like this to conditionalize it:

```yaml
- name: Check if Mesos leader service definition exists
  stat: path=/etc/init/mesos-master.conf
  register: mesos_leader_service

- name: Stop Mesos leader service
  service: name=mesos-master state=stopped
  when: mesos_leader_service.stat.exists
```